tests/test_dirty_local.py

```diff
@@ -214,3 +214,106 @@ def test_update_with_gpg_sign(
     gitconfig.set({"user.signinkey": "123456", "commit.gpgsign": "true"})
     with pytest.warns(DirtyLocalWarning):
         run_update(dst, defaults=True, overwrite=True)
+
+
+@pytest.fixture
+def setup_parallel_projects(
```

Comment:
I think it's more intuitive to name this fixture in noun form and not in imperative mood, as you're accessing the resources created by this fixture later.


```suggestion
def parallel_projects(
```